Makefile

```diff
@@ -73,6 +77,48 @@ build-tf2-cpu:
 		-t $(NGC_REGISTRY)/$(CPU_TF2_ENVIRONMENT_NAME)-$(VERSION) \
 		.
 
+.PHONY: build-tf25-cpu
+build-tf25-cpu:
+	docker build -f Dockerfile-base-cpu \
+		--build-arg BASE_IMAGE="ubuntu:18.04" \
+		--build-arg PYTHON_VERSION="3.7.10" \
+		-t $(DOCKERHUB_REGISTRY)/$(CPU_TF25_BASE_NAME)-$(SHORT_GIT_HASH) \
+		-t $(DOCKERHUB_REGISTRY)/$(CPU_TF25_BASE_NAME)-$(VERSION) \
+		.
+	docker build -f Dockerfile-default-cpu \
+		--build-arg BASE_IMAGE="$(DOCKERHUB_REGISTRY)/$(CPU_TF25_BASE_NAME)-$(SHORT_GIT_HASH)" \
+		--build-arg TENSORFLOW_PIP="tensorflow-cpu==2.5.1" \
+		--build-arg TORCH_PIP="torch==1.7.1 -f https://download.pytorch.org/whl/cpu/torch_stable.html" \
+		--build-arg TORCHVISION_PIP="torchvision==0.8.2 -f https://download.pytorch.org/whl/cpu/torch_stable.html" \
+		--build-arg LIGHTNING_PIP="pytorch_lightning==1.2.0" \
```

Comment:
should it be 
```suggestion
		--build-arg TORCH_PIP="torch==1.9 -f https://download.pytorch.org/whl/cpu/torch_stable.html" \
		--build-arg TORCHVISION_PIP="torchvision==0.10.0 -f https://download.pytorch.org/whl/cpu/torch_stable.html" \
		--build-arg LIGHTNING_PIP="pytorch_lightning==1.3.5" \
```
?

Reply:
torch 1.9 doesn't work on cuda 11.2, it has some odd crashes in some of our tests. Me and @azhou-determined didn't have luck in debugging that. So we've got to use CUDA 11.1 for it (which is what we use in tf-2.4 images). torch 1.7 works fine.

But for tf 2.5 and tf 2.6 we need cuda 11.2, so we have to use it, and as a result these "new" tensorflow images end up using "legacy" torch.